### DISCUSSION TITLE
copy and paste from pc to putty browser

### DISCUSSION THREAD

**Discussion by Simonz82** [jlesage/docker-putty#13] [Q&A] [answered]
Hi,
how can i copy text from my pc and paste it into putty?
i tried using the clipboard, the text is pasted but then how do i paste it into putty?
why is the clip to windows flag greyed out and i can't use it?
thanks
Simone

![2024-08-20_115205](https://github.com/user-attachments/assets/561d8ed9-2c91-4912-a0fd-4b81710e9bdb)

**Comment by jlesage** [comment, marked as answer]
PuTTY supports a few options for pasting. See http://the.earth.li/~sgtatham/putty/0.60/htmldoc/Chapter3.html#using-selection

The "Clip to Window" option applies only when the scaling mode is set to "None".

**Comment by Simonz82** [comment]
OK, thanks
I solved it with the middle button of my mouse.
